modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase

```diff
@@ -160,12 +177,43 @@ definition {
 	}
 
 	@description = "This is a test for COMMERCE-5807. EditSkuAndDelete"
-	@ignore = "true"
 	@priority = "3"
 	test EditSkuAndDelete {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-5807 EditSkuAndDelete
+		CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Products");
+
+		CommerceEntry.gotoMenuTab(menuTab = "SKUs");
+
+		CommerceNavigator.searchEntry(entryName = "MIN93017");
+
+		CommerceNavigator.gotoEntry(entryName = "MIN93017");
+
+		SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
+
+		Type(
+			locator1 = "CommerceEntry#SKU_FIELD",
+			value1 = "MIN930178");
+
+		Button.clickPublish();
+
+		AssertVisible(locator1 = "Message#SUCCESS");
+
+		Click(locator1 = "Icon#CLOSE");
+
+		Refresh();
+
+		CommerceNavigator.searchEntry(entryName = "MIN930178");
+
+		if (IsElementPresent(locator1 = "//a[contains(text(),'MIN930178')]")) {
```

Comment:
For the locator i think is possible to use CommerceNavigation#ANY_ENTRY_NAME (//a[contains(.,'${key_entry}')]) with key_entry = MIN930178

Reply:
Wouldn't that add more code and make it messier? 'MIN930178' is the only entry on the page due to searching for it above.

Reply:
That locator is already exists and is better to reuse it if is possible

Reply:
Changed locator.